### PR TITLE
Align tables columns in the machine details

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added bespoke styling to the tables in the machine details as maas-ui does. 

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Go to the model details
- Check the tables match the tables visible in [the design](https://app.zeplin.io/project/5d2dfd8e5b67f14c5e1d2aef/screen/5e67bbca397a5b129eb35110)

## Details
Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/424
Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/426

## Screenshots
Screenshot_2020-05-28 JAAS Dashboard

### CODE REVIEW

File: src/scss/custom/_tables.scss [hunk 1]
```diff
@@ -14,3 +14,107 @@ th,
 td {
   min-width: 150px;
 }
+
+.model-details__apps {
```
Comment (hatched) [inline src/scss/custom/_tables.scss:18]: I think that all of these changes should probably be in https://github.com/canonical-web-and-design/jaas-dashboard/blob/master/src/pages/Models/Details/_model-details.scss instead. This move should also resolve the stylelint issue.

**Comment by barrymcgee** [comment]
The third and fourth columns of App and Unit tables are unreadable at <1400px

This is why I have resisted this approach thus far, it gets very messy, very quickly. 

Screenshot 2020-05-28 at 08 39 55

**Comment by spencerbygraves** [comment]
Guess it's a -1 based on the images above.

**Comment by anthonydillon** [comment]
@spencerbygraves new screenshot attached to the description of the PR

**Comment by spencerbygraves** [comment]
@anthonydillon thanks

**Comment by anthonydillon** [comment]
@hatched @barrymcgee please do a good look through the different views as the styles were leaking out all over and I have refactored parts of the model listing.

Drive by's:
- Fixes the horizontal scrolling in the app
- Made the model-details__config component generic
- Add mixins the to at-rules for stylelint
- Moved the `p-filter` styling into a single place
- Made the model title a strong element instead of a div with bold styling
- Added message to the column headers in the relation table
- Replace the bespoke truncate class with the Vanilla one.

**Comment by hatched** [comment]
Some tiny qa issues with the new truncated values.

The status column has the value "Unknown" truncated at 1679px.
Screen Shot 2020-05-28 at 9 17 25 PM

IP addresses are truncated at the minimum width but there isn't a tooltip to get the actual value.
Statuses are truncated without an ellipsis and are also no tooltip.
Screen Shot 2020-05-28 at 9 22 23 PM

Relation provider doesn't have a tooltip when truncated
Screen Shot 2020-05-28 at 9 25 06 PM

Truncated unit names are missing tooltips
Screen Shot 2020-05-28 at 9 27 25 PM

**Comment by anthonydillon** [comment]
@hatched thanks, I noticed these too and plan to fix them in follow ups.

**Pull request merged**